held/app/src/main/java/com/held/activity/VerificationActivity.java

```diff
@@ -159,34 +163,39 @@ public void afterTextChanged(Editable editable) {
     }
 
     private void callVerificationApi() {
-        HeldService.getService().verifyUser(mPhoneNo, mPin, new Callback<VerificationResponse>() {
+        HeldService.getService().verifyUser(mAuth,mRegId, mPin,"", new Callback<VerificationResponse>() {
                     @Override
                     public void success(VerificationResponse verificationResponse, Response response) {
-                        if (verificationResponse.isVerified()) {
```

Comment:
you have skipped the isVerified() call.  how will you know if verification is successful?